<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonWidget</class>
 <widget class="QWidget" name="PersonWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ssnLabel">
       <property name="text">
        <string>SSN:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ssnEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="jobLabel">
       <property name="text">
        <string>Job:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="jobEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="scoreLabel">
       <property name="text">
        <string>Score:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="scoreCombo">
       <item><property name="text"><string>A1</string></property></item>
       <item><property name="text"><string>A2</string></property></item>
       <item><property name="text"><string>A3</string></property></item>
       <item><property name="text"><string>B1</string></property></item>
       <item><property name="text"><string>B2</string></property></item>
       <item><property name="text"><string>B3</string></property></item>
       <item><property name="text"><string>C1</string></property></item>
       <item><property name="text"><string>C2</string></property></item>
       <item><property name="text"><string>C3</string></property></item>
       <item><property name="text"><string>D1</string></property></item>
       <item><property name="text"><string>D2</string></property></item>
       <item><property name="text"><string>D3</string></property></item>
       <item><property name="text"><string>E1</string></property></item>
       <item><property name="text"><string>E2</string></property></item>
       <item><property name="text"><string>E3</string></property></item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add Person</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchEdit">
     <property name="placeholderText">
      <string>Search by name...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
